MapboxNavigation/NavigationViewController.swift

```diff
@@ -323,6 +323,11 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
         routeController.resume()
         
         styles?.forEach { $0.apply() }
+        
+        if routeController.locationManager is SimulatedLocationManager {
+            let title = NSLocalizedString("USER_IN_SIMULATION_MODE", bundle: .mapboxNavigation, value: "Simulating Navigation", comment: "Inform the user they are in simulation mode")
```

Comment:
The comment (and the one in the .strings file) should ideally be detailed enough that the translator can find the string in the UI if necessary:

> The text of a banner that appears during turn-by-turn navigation when route simulation is enabled.